hudi-common/src/test/java/org/apache/hudi/common/util/TestObjectSizeCalculator.java

```diff
@@ -73,7 +74,8 @@ public void testGetObjectSize() {
     assertEquals(32, getObjectSize(emptyClass));
     assertEquals(40, getObjectSize(stringClass));
     assertEquals(40, getObjectSize(payloadClass));
-    assertEquals(1240, getObjectSize(Schema.create(Schema.Type.STRING)));
+    assertEquals(HoodieAvroUtils.gteqAvro1_10() ? 1320 : 1240,
```

Comment:
please comment the avro pr to explain this change.

Reply:
done